lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -266,8 +266,8 @@ class MockTimers {
       );
     }
 
-    // Add 1ms to fix flaky tests
-    this.#now += (time += 1);
+    // Add 50ms to fix flaky tests
+    this.#now += time + 50;
```

Comment:
seems kinda hacky to add production code just to satisfy tests :-/

Reply:
yes, that was reverted

Reply:
I this is outdated!

Reply:
whoops, i clicked the github notification and it took me here :-) resolve away :-p